components/Footer.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import {
+  faFacebookF,
+  faTwitter,
+  faGithub,
+  faDribbble,
+} from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Footer: React.FC = () => {
+  return (
+    <footer
+      className="bg-gray-100 py-10 md:px-32"
+      style={{ backgroundImage: "url('/assets/images/jpg/footer-2.jpg')" }}
```

Comment:
Don't use absolute path's in your code base. Rather create a folder for constants, import your images and use the export image here.

Example:
import BACKGROUND from "./assets/images/jg/footer-2.jpg"

export {BACKGROUND}